etc/kayobe/ansible/templates/wazuh-secrets.yml.j2

```diff
@@ -3,12 +3,12 @@
 # Store these securely and use lookups here
 secrets_wazuh:
   # Wazuh agent authd pass
-  authd_pass: "{{ secrets_wazuh.authd_pass | default(lookup('password', '/dev/null'), true) }}"
+  authd_pass: '{{ secrets_wazuh.authd_pass | default(lookup('password', '/dev/null'), true) }}'
```

Comment:
`default(lookup('password', '/dev/null')` the single quotes in here will need replacing with doubles now. Same for the rest of the file

Reply:
Good spot. Not sure why that didn't come up when I tested it

Reply:
Retested and same result. I guess it's cause it's a proper jinja template not an ansible playbook, so it parses differently. I'll update it anyway for consistency

Reply:
Done

Reply:
Funky that the behaviour differs between inline and separate templates